src/main/io/osd.h

```diff
@@ -81,6 +81,8 @@ typedef enum {
     OSD_MAP_TAKEOFF,
     OSD_RADAR,
     OSD_DEBUG, // Number 46. Intentionally absent from configurator and CMS. Set it from CLI.
```

Comment:
Shouldn't OSD_DEBUG be last ?

Reply:
(we discussed this, but let's add it here for others)

OSD_DEBUG is just an OSD indicator that we don't show in CMS nor in the configurator. From the firmware PoV there's nothing special about it.